tile/TileSeparateCrossbarRTL.py

```diff
@@ -123,7 +124,7 @@ def construct(s, DataType, PredicateType, CtrlPktType, CtrlSignalType,
         s.to_mem_wdata //= s.element.to_mem_wdata[i]
       else:
         s.element.to_mem_raddr[i].rdy //= 0

```

Comment:
@yo96 Is it true that now we can simply assign `0` to it? I remember we need `//= b1(0)` before due to pymtl limitations?

Reply:
Yes, we can. PyMTL can automatically infer the bitwidth of a constant.